src/libraries/System.Drawing.Common/src/System/Drawing/DrawingComWrappers.cs

```diff
@@ -0,0 +1,283 @@
+
+                inst.CopyTo(pstmStream, cb, pcbRead, pcbWritten);
+                return Interop.HRESULT.S_OK;
+            }
+
+            [UnmanagedCallersOnly]
+            private static Interop.HRESULT CommitImplementation(IntPtr thisPtr, uint grfCommitFlags)
+            {
+                Interop.Ole32.IStreamComWrapper inst = ComInterfaceDispatch.GetInstance<Interop.Ole32.IStreamComWrapper>((ComInterfaceDispatch*)thisPtr);
+                inst.Commit(grfCommitFlags);
+                return Interop.HRESULT.S_OK;
+            }
+
+            [UnmanagedCallersOnly]
+            private static Interop.HRESULT RevertImplementation(IntPtr thisPtr)
+            {
+                Interop.Ole32.IStreamComWrapper inst = ComInterfaceDispatch.GetInstance<Interop.Ole32.IStreamComWrapper>((ComInterfaceDispatch*)thisPtr);
+                inst.Revert();
+                return Interop.HRESULT.S_OK;
+            }
+
+            [UnmanagedCallersOnly]
+            private static Interop.HRESULT LockRegionImplementation(IntPtr thisPtr, ulong libOffset, ulong cb, uint dwLockType)
+            {
+                Interop.Ole32.IStreamComWrapper inst = ComInterfaceDispatch.GetInstance<Interop.Ole32.IStreamComWrapper>((ComInterfaceDispatch*)thisPtr);
+                return inst.LockRegion(libOffset, cb, dwLockType);
+            }
+
+            [UnmanagedCallersOnly]
+            private static Interop.HRESULT UnlockRegionImplementation(IntPtr thisPtr, ulong libOffset, ulong cb, uint dwLockType)
+            {
+                Interop.Ole32.IStreamComWrapper inst = ComInterfaceDispatch.GetInstance<Interop.Ole32.IStreamComWrapper>((ComInterfaceDispatch*)thisPtr);
+                return inst.UnlockRegion(libOffset, cb, dwLockType);
+            }
+
+            [UnmanagedCallersOnly]
+            private static Interop.HRESULT StatImplementation(IntPtr thisPtr, out Interop.Ole32.STATSTG pstatstg, Interop.Ole32.STATFLAG grfStatFlag)
+            {
+                Interop.Ole32.IStreamComWrapper inst = ComInterfaceDispatch.GetInstance<Interop.Ole32.IStreamComWrapper>((ComInterfaceDispatch*)thisPtr);
+                inst.Stat(out pstatstg, grfStatFlag);
+                return Interop.HRESULT.S_OK;
+            }
+
+            [UnmanagedCallersOnly]
+            private static Interop.HRESULT CloneImplementation(IntPtr thisPtr, IntPtr* ppstm)
+            {
+                if (ppstm == null)
+                {
+                    return Interop.HRESULT.STG_E_INVALIDPOINTER;
+                }
+
+                Interop.Ole32.IStreamComWrapper inst = ComInterfaceDispatch.GetInstance<Interop.Ole32.IStreamComWrapper>((ComInterfaceDispatch*)thisPtr);
+
+                *ppstm = Instance.GetOrCreateComInterfaceForObject(inst.Clone(), CreateComInterfaceFlags.None);
+                return Interop.HRESULT.S_OK;
+            }
+        }
+
+#pragma warning disable CS0649 // fields are never assigned to
+        internal struct IPictureVtbl
```

Comment:
Entirely up to you, but this type is just extra metadata.

Reply:
Same for `IStreamVtbl` fields.

Reply:
Same for `IUnknownImpl` entirely.

Reply:
I've removed them.